src/Host.h

```diff
@@ -438,6 +439,7 @@ class Host : public QObject
     void setCommandLineHistorySaveSize(const int lines);
     bool showIdsInEditor() const { return mShowIDsInEditor; }
     void setShowIdsInEditor(const bool isShown) { mShowIDsInEditor = isShown; if (mpEditorDialog) {mpEditorDialog->showIDLabels(isShown);} }
+    QNetworkAccessManager* getNetworkAccessManager() const { return mpNetworkAccessManager; }
```

Comment:
If you are having this `public` "getter" - then perhaps the underlying `mpNetworkAccessManager` instance should be `private` so other classes can't get direct unfettered access to it?